config/routes.php

```diff
@@ -154,6 +155,7 @@ function (RouteCollector $route): void {
                 $route->get('/users/{user_id:(?:\d+|self)}', 'Api\UsersController@user');
                 $route->get('/users/{user_id:(?:\d+|self)}/angeltypes', 'Api\AngelTypeController@ofUser');
                 $route->get('/users/{user_id:(?:\d+|self)}/shifts', 'Api\ShiftsController@entriesByUser');
+                $route->get('/users/{user_id:(?:\d+|self)}/worklogs', 'Api\UsersController@workLogs');
```

Comment:
```suggestion
                $route->get('/users/{user_id:(?:\d+|self)}/worklogs', 'Api\UsersController@worklogs');
```
Everywhere else it's "Worklog"